### PR TITLE
Add Eclipselink 4.0's JSON bundle for JSON support, and FAT.

### PR DESCRIPTION
**Pull request by jgrassel**
Added eclipselink json bundle, and FAT test case to verify it.  Also updating to use Eclipselink 4.0.0-RC1 instead of the local built version of eclipselink.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
#build #run-libby-bot

**Review by brideck — APPROVED** [review]

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
#build #run-libby-bot

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
#build #run-libby-bot

**Comment by jgrassel** [comment]
#build #run-libby-bot

File: dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd [hunk 1]
```diff
@@ -112,22 +118,41 @@ DynamicImport-Package: \
...
+    implementation:=com.ibm.wsspi.classloading.ResourceProvider; \
+    provide:=com.ibm.wsspi.classloading.ResourceProvider; \
+    configuration-policy:=ignore; \
+    properties:="resources=${app-resources}"
+
 -buildpath: \
-    io.openliberty.org.eclipse.persistence:org.eclipse.persistence.core;version="[4.0.0,4.1.0)",\
-    io.openliberty.org.eclipse.persistence:org.eclipse.persistence.jpa;version="[4.0.0,4.1.0)",\
+    org.eclipse.persistence:org.eclipse.persistence.core;version=${eclVersion};maven-version=${eclVersion}-RC2,\
+    org.eclipse.persistence:org.eclipse.persistence.jpa;version=${eclVersion};maven-version=${eclVersion}-RC2,\
```
Comment (dazey3) [inline dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd]: Maybe try
```
    org.eclipse.persistence:org.eclipse.persistence.core;version=${eclFullVersion};strategy=exact,\
    org.eclipse.persistence:org.eclipse.persistence.jpa;version=${eclFullVersion};strategy=exact,\
```

File: dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd [hunk 2]
```diff
@@ -112,22 +118,41 @@ DynamicImport-Package: \
...
   @${repo;org.eclipse.persistence:org.eclipse.persistence.asm;${asmFullVersion};EXACT}!/!META-INF/maven/*,\
-  @${repo;io.openliberty.org.eclipse.persistence:org.eclipse.persistence.core;${eclVersion}}!/!META-INF/maven/*,\
-  @${repo;io.openliberty.org.eclipse.persistence:org.eclipse.persistence.jpa;${eclVersion}}!/!META-INF/maven/*,\
-  @${repo;io.openliberty.org.eclipse.persistence:org.eclipse.persistence.jpa.jpql;${eclVersion}}!/!META-INF/maven/*,\
-  @${repo;io.openliberty.org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen.processor;${eclVersion}}!/!META-INF/maven/*
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.core;${eclVersion}}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa;${eclVersion}}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.jpql;${eclVersion}}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen.processor;${eclVersion}}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.json;${eclVersion}}!/!META-INF/maven/*
```
Comment (dazey3) [inline dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd]: Maybe try:
```
  @${repo;org.eclipse.persistence:org.eclipse.persistence.core;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.jpql;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen.processor;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
  @${repo;org.eclipse.persistence:org.eclipse.persistence.json;${eclFullVersion};EXACT}!/!META-INF/maven/*
```

File: dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd [hunk 3]
```diff
@@ -13,7 +13,7 @@ bVersion=1.0
 
 # Eclipselink version info
 eclVersion=4.0.0
-eclFullVersion=${eclVersion}.202d992
+eclFullVersion=${eclVersion}-RC2
```
Comment (dazey3) [inline dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd]: Maybe try:
```
eclFullVersion=${eclVersion}.RC2
```

**Comment by jgrassel** [comment]
#build #run-libby-bot

File: dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd (hunk 3 above)
Comment (jgrassel) [reply dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd]: Yeah, finally got this to work.  Thanks!

**Comment by jgrassel** [comment]
#run-libby-bot

**Pull request merged**